app/assets/javascripts/app/project/edit.js

```diff
@@ -29,3 +35,40 @@ App.addChild('ProjectEdit', _.extend({
   }
 }, Skull.Tabs));
 
+// FIXME: MORE DRY HERE !!
```

Comment:
I believe we should use the ProjectEdit for everything. Just call all the this.route